### PR TITLE
Feat/playlist

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
1. user can create a playlist from video listing page
2. add video to a playlist
3. can see all the playlist on playlist page
4. remove playlist from playlist page
5. create new playlist from playlist page also

### CODE REVIEW

**Review by rutvikpumak — APPROVED** [review]
Code looks clean.
Playlist functionality working expected
Good to merge

File: src/component/navbar/index.jsx [hunk 1]
```diff
@@ -7,20 +7,37 @@ import AccountCircleOutlinedIcon from '@mui/icons-material/AccountCircleOutlined
 import LogoutIcon from '@mui/icons-material/Logout';
 import MenuIcon from '@mui/icons-material/Menu';
 import SearchIcon from '@mui/icons-material/Search';
+import CloseIcon from '@mui/icons-material/Close';
```
Comment (ivinayakg) [inline src/component/navbar/index.jsx:10]: Just a suggestion, this is absolutely correct, but what you can do is create a icons. Js file and then export all the icons you need in your app from there

**Review by ivinayakg — APPROVED** [review]
Great works, really enjoyed going through, I saw a lot of new techniques.

File: src/service/playlist-service.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+
```
Comment (ivinayakg) [inline src/service/playlist-service.js:2]: If you have added this file then make sure all the api calls are in this file, as if not may lead to confusion then.

**Review by ivinayakg — APPROVED** [review]

**Comment by SHIKHASINGH1506** [comment]
> Great works, really enjoyed going through, I saw a lot of new techniques.

Thanks for the review!

**Pull request merged**